### PR TITLE
Remove any reference for GH issue #332  (DEV)

### PR DESCRIPTION
**Pull request by mtwalli**
Update localisation points in docs to reflect the current process

### CODE REVIEW

**Comment by mtwalli** [comment]
@MikeMcC399 Please have a look at this PR?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -17,9 +17,7 @@ __Thank you for this PR! Please consider the following:__
     * If this PR comes from a
       fork, please [allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
     * The PR can not contain changes in localization files, e.g. `values-[LANGUAGE-CODE]/strings/*`
-      and `/`
-      or `assets/*` (see [#332](https://github.com/corona-warn-app/cwa-app-android/issues/332) for
-      further information).
+      and `/` or `assets/*`). Localizations are usually provided and approved by UA team.
```
Comment (MikeMcC399) [inline .github/pull_request_template.md:20]:
```suggestion
      and `/` or `assets/*`). Localizations are usually provided and approved by the UA team.
```

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ The goal of this project is to develop the official Corona-Warn-App for Germany
 * In General every TODO comment within the code or the documentation can be regarded as an issue. You are free to tackle the TODOs anytime!
 * We are aware of the Play Store Limitations of GMS.
 * Without your own server instance (e.g. a local Docker setup), you will not be able to test the Application. For help, please refer to the [server implementation](https://github.com/corona-warn-app/cwa-server).
-* Strings including translations are handled separately, for suggestions / findings please have a look at issue [#332](https://github.com/corona-warn-app/cwa-app-android/issues/332)
+* Strings including translations are provided and approved by UA team, for suggestions / findings please open a dedicated issue for that)
```
Comment (MikeMcC399) [inline README.md:43]:
```suggestion
* Strings including translations are provided and approved by the UA team. For suggestions / findings please open a dedicated issue.
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Please see in-line suggestions.

**Review by MikeMcC399 — APPROVED** [review]
LGTM!

**Pull request merged**